helm: add cephFileSystems storageClass.pool (#9838)
rook/rook: Documentation/helm-ceph-cluster.md

| `spec`                       | The CephFileSystem spec, see the [CephFilesystem CRD](ceph-filesystem-crd.md) documentation.                                                                | see values.yaml   |
| `storageClass.enabled`       | Whether a storage class is deployed alongside the CephFileSystem                                                                                            | `true`            |
| `storageClass.name`          | The name of the storage class                                                                                                                               | `ceph-filesystem` |

> | `storageClass.pool`          | The name of [Data Pool](ceph-filesystem-crd.md#pools), the final pool to be used: `<name>-<storageClass.pool>`                                              | `data0`           |

Reviewer: Perhaps this answers my other question below?
```suggestion
| `storageClass.pool`          | The name of [Data Pool](ceph-filesystem-crd.md#pools), without the filesystem name prefix                                             | `data0`           |
```
